### DISCUSSION TITLE
What data formats should Studio support?

### DISCUSSION THREAD

**Discussion by esthersweon** [foxglove/community#3] [Q&A]
Let us know what data format you're using in your robotics project, so we can build out Studio support for it!

**Comment by phonymacoroni** [comment]
bgra8 for images would be awesome!

**Reply by jtbandes** [reply]
Great suggestion! Should be relatively straightforward to add since it's similar to `bgr8` which we already support. Do you happen to have a sample bag with some bgra8 images we can use to test with?

I filed a ticket https://github.com/foxglove/studio/issues/1273 – if you have sample data to share, please attach it there!

By the way I saw in your profile that you've worked on WPILib! A couple of us at Foxglove are FRC alumni and I think we would be excited to explore integrations with Studio in the future 😄

**Reply by phonymacoroni** [reply]
Thank you very much! 
Awesome to hear it! I will be sure to pass the message along :)
I am loving exploring Foxglove, really excited to get into visualization markers as well. Thank you all for the awesome work!
